TrkReco/src/RobustHelixFit.cc

```diff
@@ -325,7 +325,7 @@ namespace mu2e
 	if ( facezF1 == facezF2 )  continue;
 
 	float dz = hitP2->pos().z() - hitP1->pos().z();//facezF2->z - facezF1->z;
-	if (dz < _minzsep || dz > _maxzsep)          continue;
```

Comment:
There is also a statement dz < _minzsep on line 766 of this file, shouldn't that get fixed as well?

Reply:
I guess you are right! I didn't notice it because my students debugged only the initFZ function.

Reply:
Hi Dave,

here you can find a few plots I just made with the CE+mix sample. The blue line is my repository before I made the change you suggested, while the dots represent the results after I made the change.
https://drive.google.com/drive/folders/1vw7Hg76buFwTz3EKAmjS3U3hbUAKEQbe?usp=sharing

Let me know if you want additional plots.

Cheers,

Giani